includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -51,7 +51,6 @@ public function __construct( $args = [] ) {
 	 */
 	public function register_embed() {
 		add_filter( 'embed_oembed_html', [ $this, 'filter_embed_oembed_html' ], 10, 2 );
-		add_shortcode( 'youtube', [ $this, 'shortcode' ] ); // @todo Deprecated. See <https://github.com/ampproject/amp-wp/issues/3309>.
 		add_filter( 'wp_video_shortcode_override', [ $this, 'video_override' ], 10, 2 );
```

Comment:
This `video_override` method needs to be restored.

Reply:
Ah, good point. I'll do that.

Reply:
It's restored in https://github.com/ampproject/amp-wp/pull/3678/commits/509704014bca92220b4c7240faf213981357878c